ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -88,22 +88,37 @@
 
 #### Router OSPF Router Redistribution
 
-| Process ID | Source Protocol | Route Map |
-| ---------- | --------------- | --------- |
+| Process ID | Source Protocol | Include Leaked | Route Map |
+| ---------- | --------------- | -------------- | --------- |
 {%         for process_id in router_ospf.process_ids | arista.avd.natural_sort('id') %}
 {%             if process_id.redistribute is arista.avd.defined %}
 {%                 set source_protocols = [] %}
 {%                 if process_id.redistribute.connected is defined %}
```

Comment:
Actually can we change this line to `arista.avd.defined` as well? (I know it is not part of your PR but well this should be refactored so let's take the opportunity)

Reply:
As discussed, `is defined` must be kept for now since `connected:` can be `null`.